ibm/data_source_ibm_event_streams_topic.go

```diff
@@ -67,6 +66,7 @@ func dataSourceIBMEventStreamsTopicRead(d *schema.ResourceData, meta interface{}
 			return nil
 		}
 	}
-	log.Printf("[DEBUG]dataSourceIBMEventStreamsTopicRead topic %s does not exist", topicName)
-	return fmt.Errorf("topic %s does not exist", topicName)
```

Comment:
For datasource if topic doesn't exist we should throw the error the earlier good was correct